HelpMyStreetFE/HelpMyStreetFE/Views/Shared/DisplayTemplates/RequestSummaryViewModel.cshtml

```diff
@@ -16,8 +16,25 @@
     bool singleJob = Model.Item.JobBasics.Count() == 1 && Model.Item.RequestType.Equals(RequestType.Task);
     var jobId = singleJob ? Base64Utils.Base64Encode(Model.Item.JobBasics.First().JobID) : "";
 
+    var hmsRef = Model.Item.JobSummaries.First().HmsReference.Split("-");
```

Comment:
We should add HmsReference to the RequestSummary class instead of working it out here.